radio/src/gui/480x272/radio_calibration.cpp

```diff
@@ -39,14 +39,14 @@ enum CalibrationState {
 
 void drawStick(coord_t x, coord_t y, const BitmapBuffer * background, int16_t xval, int16_t yval)
 {
-  static const BitmapBuffer * stick = BitmapBuffer::load(getThemePath("stick_pointer.png"));
+  const BitmapBuffer * stick = BitmapBuffer::load(getThemePath("stick_pointer.png"));
```

Comment:
This is not good, the file is loaded every time the LCD is refreshed. And delete is also missing - memory leak.